pkgs/servers/monitoring/prometheus/smartctl-exporter/default.nix

```diff
@@ -24,7 +24,12 @@ buildGoModule rec {
   passthru.tests = { inherit (nixosTests.prometheus-exporters) smartctl; };
 
   meta = with lib; {
-    description = "Export smartctl statistics for Prometheus";
+    description = ''
+      Export smartctl statistics for Prometheus
+
+      To fix permissions for scanning NVMe devices, add a similar udev rule to <literal>services.udev.extraRules</literal>:
+      <literal>SUBSYSTEM=="nvme", KERNEL=="nvme[0-9]*", GROUP="disk"</literal>
```

Comment:
I think we should just set up the rule if the exporter gets enabled, if it makes auto-discovery work properly.

Reply:
Ah, we're in that funny exporter module structure. Yeah, I'm clueless as well.

Reply:
I'll create a separate PR to discover options if you don't mind